### PR TITLE
Send notifications to connected runctl clients

### PR DESCRIPTION
**Pull request by rmg**
Any client connected to us via TCP should also be "subscribed" to
cluster/app related events.

### CODE REVIEW

**Comment by sam-github** [comment]
This "send to both" approach I don't think is going to work. https://github.com/strongloop/strong-supervisor/blob/master/lib/metrics.js#L116  and traces, and slow endpoints all send explicitly to the parentCtl channel, but they need to be sending notifications to something that can then send it to either the parentCtl or a runctl channel, right? I think you are getting status notifications now, but no the other ones.

**Comment by sam-github** [comment]
by send-to-both, I mean doing ipcctl.notify() and runctl.notify().

**Comment by rmg** [comment]
@sam-github refactored into a wrapper that dispatches to both of the runctl channels.

File: test/metrics.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ describe('metrics', function() {
     delete process.env.STRONGLOOP_METRICS;
   });
 
-  it('returns false when STRONGLOOP_METRICS not in env', function() {
+  it.skip('returns false when STRONGLOOP_METRICS not in env', function() {
```
Comment (sam-github) [inline test/metrics.js]: I think you can just delete this test, not skip it.

**Comment by sam-github** [comment]
LGTM

File: test/metrics.js (hunk 1 above)
Comment (rmg) [inline test/metrics.js]: Will do. Soliciting a comment was pretty much the purpose of the change ;-)

**Pull request merged**